neo.UnitTests/TestBlockchain.cs

```diff
@@ -9,6 +9,9 @@ namespace Neo.UnitTests
 {
     public static class TestBlockchain
     {
+        // standard time: 2 seconds
+        public static uint BlockTime = 2;
```

Comment:
We can use `Blockchain.SecondsPerBlock` directly.

Reply:
That's true! Value was already read by ProtocolSettings. I'll fix this.

Reply:
Ok Erik, I'm using `Blockchain.SecondsPerBlock` on tests now, which is more consolidated. I moved this line to ProtocolSettings and made it private, because this is the test that guarantees  `protocol.json` value is actually being read and passed to `Blockchain.SecondsPerBlock` (so we make sure it matches the expected value 2).